code/java/src/main/java/com/myorg/CdkWorkshopStack.java

```diff
@@ -0,0 +1,42 @@
+package com.myorg;
+
+import software.amazon.awscdk.App;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.samples.tableviewer.TableViewer;
+import software.amazon.awscdk.samples.tableviewer.TableViewerProps;
+import software.amazon.awscdk.services.apigateway.LambdaRestApi;
+import software.amazon.awscdk.services.apigateway.LambdaRestApiProps;
+import software.amazon.awscdk.services.lambda.Code;
+import software.amazon.awscdk.services.lambda.Function;
+import software.amazon.awscdk.services.lambda.FunctionProps;
+import software.amazon.awscdk.services.lambda.Runtime;
+
+public class CdkWorkshopStack extends Stack {
+    public CdkWorkshopStack(final App parent, final String name) {
+        this(parent, name, null);
+    }
+
+    public CdkWorkshopStack(final App parent, final String name, final StackProps props) {
+        super(parent, name, props);
+
+        Function hello = new Function(this, "HelloHandler", FunctionProps.builder()
+                .withRuntime(Runtime.NODE_J_S810)
```

Comment:
@RomainMuller can you open an issue to fix these enum names?

Reply:
awslabs/jsii/#359